src/EPR.Payment.Facade/Controllers/Payments/PaymentsController.cs

```diff
@@ -100,32 +100,32 @@ public async Task<IActionResult> InitiatePayment([FromBody] PaymentRequestDto re
         }
     }
 
-    [HttpPost("{govPayPaymentId}/complete")]
+    [HttpPost("{externalPaymentId}/complete")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Comment:
The method is marked to produce 500InternalServerError but the code does not return this response code in any condition. Remove this if not required.

Reply:
I have updated the ProducesResponseTypes for InitiatePayment and CompletePayment so they accurately reflect the possible response types.